### PR TITLE
Added Ansible UTM Info Module for proxy profile entities.

### PR DESCRIPTION
**Pull request by maunzCache**
##### SUMMARY
This PR will add a new utm info module which allows the management of proxy profile entries of the Sophos UTM. This module integrates with the existing utm_utils .

##### ISSUE TYPE
* New Module Pull Request

##### COMPONENT NAME
* module/web_infrastructure/sophos_utm/utm_proxy_profile_info.py

### CODE REVIEW

**Comment by getjack** [comment]
@stearz @RickS-C137 Please help out reviewing :)

**Review by MatrixCrawler — APPROVED** [review]

**Comment by MatrixCrawler** [comment]
shipit

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = """
+---
+module: utm_proxy_profile_info
+
+author:
+    - Stephan Schwarz (@stearz)
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py:21]: @getjack This should be your details here

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        description:
+          - The name of the object. Will be used to identify the entry
+        required: true
+
+extends_documentation_fragment:
+    - utm
+"""
+
+EXAMPLES = """
+- name: Reading UTM proxy_profile
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py:42]: Is this a full example, would you need to `register` the result (info) returned?

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py [hunk 3]
```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
```
Comment (gundalow) [inline lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py:1]: I wonder if this would be better as a facts module?

**Review by gundalow — CHANGES_REQUESTED** [review]
Updates needed

File: lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py (hunk 1 above)
Comment (maunzCache) [reply lib/ansible/modules/web_infrastructure/sophos_utm/utm_proxy_profile_info.py:21]: I'm not the author just commiting it for my colleague.

**Pull request closed without merging**